examples/demo/src/ewf_roc_server_execution.e

```diff
@@ -92,6 +92,9 @@ feature -- CMS setup
 
 			create {CMS_DEMO_MODULE} m.make
 			a_setup.register_module (m)
+
+			create {GOOGLE_CUSTOM_SEARCH_MODULE} m.make
```

Comment:
the module folder is "custom_search" but the  module is "google_custom_search_module"
maybe we should harmonize that.

probably a module folder "google_search" would be more pertinent.